app/models/spotlight/reindexing_log_entry.rb

```diff
@@ -0,0 +1,42 @@
+module Spotlight
+  ##
+  # a log entry representing an attempt to reindex some number of records in an exhibit
+  class ReindexingLogEntry < ActiveRecord::Base
+    enum job_status: { unstarted: 0, in_progress: 1, succeeded: 2, failed: 3 }
+
+    belongs_to :exhibit, class_name: 'Spotlight::Exhibit'
+    belongs_to :user, class_name: '::User'
+
+    # null start times sort to the top, to more easily surface pending reindexing
+    default_scope { order('start_time IS NOT NULL, start_time DESC') }
```

Comment:
TL;DR:  i think it's generic SQL, and have some citations that imply that, but i couldn't find a single solid reference confirming that it is.

references i do have:
* the stack exchange post i initially relied on:  https://stackoverflow.com/a/7055259
* sqlite: ordering term can be an expr:  https://www.sqlite.org/syntax/ordering-term.html
* mysql: similar advice in old comments on docs, but can't find syntax diagram:  https://dev.mysql.com/doc/refman/5.7/en/working-with-null.html
* postgres: "The sort expression(s) can be any expression that would be valid in the query's select list":  https://www.postgresql.org/docs/8.3/static/queries-order.html
* and seems worth making explicit, because of differing default behavior in at least mysql and oracle:
  * mysql:  "When doing an ORDER BY, NULL values are presented first if you do ORDER BY ... ASC and last if you do ORDER BY ... DESC."    https://dev.mysql.com/doc/refman/5.7/en/working-with-null.html
  * oracle:  "If the null ordering is not specified then the handling of the null values is:  ... NULLS FIRST if the sort is DESC"    https://docs.oracle.com/javadb/10.8.3.0/ref/rrefsqlj13658.html